src/types/app-deployer.ts

```diff
@@ -300,9 +301,9 @@ export class AppDeployer {
       }
       const createIndex = await composer.count()
       if ('method' in deleteParams) {
-        composer.addAppDeleteMethodCall({ appId: existingApp.appId, ...deleteParams })
+        composer.addAppDeleteMethodCall({ ...deleteParams, appId: existingApp.appId })
       } else {
-        composer.addAppDelete({ appId: existingApp.appId, ...deleteParams })
```

Comment:
Are these changes (in a few places) deliberate? In general I was pretty careful about ordering of spreading but it’s possible I got some wrong